optimize/backend/src/main/java/io/camunda/optimize/service/importing/AllEntitiesBasedImportIndexHandler.java

```diff
@@ -1,71 +0,0 @@
-/*
- * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
- * one or more contributor license agreements. See the NOTICE file distributed
- * with this work for additional information regarding copyright ownership.
- * Licensed under the Camunda License 1.0. You may not use this file
- * except in compliance with the Camunda License 1.0.
- */
-package io.camunda.optimize.service.importing;
-
-import io.camunda.optimize.dto.optimize.index.AllEntitiesBasedImportIndexDto;
-import io.camunda.optimize.service.db.reader.ImportIndexReader;
-import io.camunda.optimize.service.importing.page.AllEntitiesBasedImportPage;
-import io.camunda.optimize.service.util.DatabaseHelper;
-import io.camunda.optimize.service.util.configuration.ConfigurationService;
-import jakarta.annotation.PostConstruct;
-import java.util.Optional;
-import lombok.RequiredArgsConstructor;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.config.ConfigurableBeanFactory;
-import org.springframework.context.annotation.Scope;
-
-@RequiredArgsConstructor
-@Scope(value = ConfigurableBeanFactory.SCOPE_PROTOTYPE)
-public abstract class AllEntitiesBasedImportIndexHandler
```

Comment:
what was this used for? I am not sure I recall

Reply:
Its not super clear to me either but I [think](https://github.com/camunda/camunda/pull/20366/files/1f17a196106825cec884ae2ba6c12c0ac466d40c#discussion_r1694796587) its no longer used. I will have a second look